src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
Actually 50 attempts is not so bad because we use [Exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff) approach for timeout.
A would create disableRepeater flag to turn repeater off. Is it ok?